### PR TITLE
voyager unit test

### PR DESCRIPTION
**Pull request by simon-meng-cn**
Unit test for voyager integration. It tests all voyager artifacts (generated yaml files and kubernetes objects that are created at runtime by the voyager).

### CODE REVIEW

File: operator/pom.xml [hunk 1]
```diff
@@ -260,6 +260,11 @@
             <artifactId>client-java</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.appscode.voyager</groupId>
+            <artifactId>client-java-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```
Comment (rjeberhard) [inline operator/pom.xml]: Two problems: 1) You can't depend on a SNAPSHOT build, and 2) you need to add a test scope element.

Comment (rjeberhard) [reply operator/pom.xml]: Looks like they haven't published a release.  You will need to either not use this dependency or ask them to publish a release to maven.

Comment (lilyhe123) [reply operator/pom.xml]: have created an issue to voyager for this: https://github.com/voyager-client/java/issues/5

Comment (tamalsaha) [reply operator/pom.xml]: Looking into this.

Comment (tamalsaha) [reply operator/pom.xml]: I have released `com.appscode.voyager:client-java:0.1.0`. https://oss.sonatype.org/content/groups/public/com/appscode/voyager/client-java/ . It should be usable now. It takes around 7-8 hours to show up in https://search.maven.org/#search%7Cga%7C1%7Cappscode

Comment (simon-meng-cn) [reply operator/pom.xml]: I have updated the dependency to released version. It works fine.

**Comment by simon-meng-cn** [comment]
Voyager unit test code has been merged into branch issue_293. It can be submitted with voyager change. See pull request https://github.com/oracle/weblogic-kubernetes-operator/pull/339

**Pull request closed without merging**